src/scss/phy/my-account.scss

```
@@ -18,3 +18,14 @@
     width: max-content;
   }
 }
+
```

Nothing up with what you've done, but if we're fixing alignments here anyway then the icon in the header box on Physics has been offset and annoying me, so we might as well get this in at the same time.

```suggestion

.connect-list {
  .connect-list-header {
    .icon-person-active {
      margin-top: 7px;
    }
  
    h4 {
      padding: 0rem 1rem;
    }
  }
}

```